[openSUSE 15.1] Missing libseccomp headers

For openSUSE 15.1 which already install with following packages:
```
zypper install -y \
  autoconf \
  automake \
  cmake \
  gcc \
  gettext \
  glibc-static \
  libcap-devel \
  libseccomp-devel \
  libtool \
  libyajl-devel \
  libyajl2 \
  make \
  systemd-devel
```

After that try to compile crun with:
```
git clone https://github.com/containers/crun.git
cd crun
./autogen.sh
./configure
```

But result with following error message:
```
checking seccomp.h usability... no
checking seccomp.h presence... no
checking for seccomp.h... no
configure: error: *** Missing libseccomp headers
```

Detail error log could be found from here: https://travis-ci.com/github/alvistack/ansible-role-crun/jobs/335714655#L1096

P.S. other OS such as CentOS 8 or Fedora 32 or Debian 10 or Ubuntu 18.04/19.10/20.04 are all working fine ;-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -35,8 +35,9 @@ libcrun_la_SOURCES = src/libcrun/utils.c \
 			src/libcrun/chroot_realpath.c \
 			src/libcrun/signals.c
 
-libcrun_la_CFLAGS = -I $(abs_top_builddir)/libocispec/src -I $(abs_top_srcdir)/libocispec/src
-libcrun_la_LIBADD = libocispec/libocispec.la
+libcrun_la_CFLAGS = -I $(abs_top_builddir)/libocispec/src -I $(abs_top_srcdir)/libocispec/src $(SECCOMP_CFLAGS)
+
+libcrun_la_LIBADD = libocispec/libocispec.la $(SECCOMP_LIBS)
 
 if PYTHON_BINDINGS
 pyexec_LTLIBRARIES = python_crun.la

--- a/configure.ac
+++ b/configure.ac
@@ -43,10 +43,13 @@ dnl libseccomp
 AC_ARG_ENABLE([seccomp],
 	AS_HELP_STRING([--disable-seccomp], [Ignore libseccomp and disable support]))
 AS_IF([test "x$enable_seccomp" != "xno"], [
-	AC_CHECK_HEADERS([seccomp.h], [], [AC_MSG_ERROR([*** Missing libseccomp headers])])
-	AS_IF([test "$ac_cv_header_seccomp_h" = "yes"], [
-		AC_SEARCH_LIBS(seccomp_rule_add, [seccomp], [AC_DEFINE([HAVE_SECCOMP], 1, [Define if seccomp is available])], [AC_MSG_ERROR([*** libseccomp headers not found])])
-		AC_SEARCH_LIBS(seccomp_arch_resolve_name, [seccomp], [AC_DEFINE([SECCOMP_ARCH_RESOLVE_NAME], 1, [Define if seccomp_arch_resolve_name is available])], [ ])
+	PKG_CHECK_MODULES([SECCOMP], [libseccomp], [], [
+		AC_CHECK_HEADERS([seccomp.h], [], [AC_MSG_ERROR([*** Missing libseccomp headers])])
+		AS_IF([test "$ac_cv_header_seccomp_h" = "yes"], [
+			AC_SEARCH_LIBS(seccomp_rule_add, [seccomp], [AC_DEFINE([HAVE_SECCOMP], 1, [Define if seccomp is available])], [AC_MSG_ERROR([*** libseccomp headers not found])])
+			AC_SEARCH_LIBS(seccomp_arch_resolve_name, [seccomp], [AC_DEFINE([SECCOMP_ARCH_RESOLVE_NAME], 1, [Define if seccomp_arch_resolve_name is available])], [ ])
+		])
+		AC_SUBST([SECCOMP_LIBS], [-lseccomp])
 	])
 ])
 

